spec/models/recommended_link_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe RecommendedLink, 'validations' do
+  it 'is invalid without a title attribute' do
+    attributes = FactoryGirl.attributes_for(:recommended_link, title: nil)
+
+    expect(new_recommended_link_with(attributes)).not_to be_valid
+  end
+
+  it "is invalid with a duplicate link" do
+    FactoryGirl.create(:recommended_link, title: 'Tax', link: 'https://www.tax.service.gov.uk/', description: 'Self assessment', keywords: 'self, assessment, tax', search_index: 'mainstream')
```

Comment:
you can drop the `FactoryGirl.` from all these factory_girl methods. Cleans the spec up a bit.